objects/authentication_factor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
```

Comment:
`nit`: I might make this description even more general.  We're intending to use it for recording MFA, but functionally, it just represents an authentication factor.  We're very use to first factors being passwords, but they don't have to be, and could be recorded using this class too, in theory!  In python pseudocode, I might imagine this as:

```py
class MultiFactorAuthentication:
    first_factor: AuthenticationFactor
    second_factor: AuthenticationFactor
```

Hopefully that helps clarify the way I'm thinking about this?

Reply:
Totally makes sense! I updated the description to be more general like this, let me know what you think about the update